### PR TITLE
sys-fs/cryfs: bump to 0.10.3

### PR DESCRIPTION
**Pull request by DankDumpster**

### CODE REVIEW

**Comment by thesamesam** [comment]
Thanks. Can you commit with 'repoman commit' or 'pkgdev commit' to ensure that the commit message is right?

Also, we avoid 'git mv' in general because we need to keep the old version around. The new version needs to be committed with unstable keywords (```ekeyword ~all foo.ebuild```).

**Comment by DankDumpster** [comment]
Sorry for that, it should be fixed now! :smile:

**Comment by DankDumpster** [comment]
@thesamesam could you take a look?

**Comment by thesamesam** [comment]
Looking good!

two questions:
1. If the patches are the same (not checked), no need to copy them, just change the reference in the new ebuild
2. 2. Can you add the GCO sign off with your name?

**Comment by DankDumpster** [comment]
Sorry forgot the gco sign off

**Comment by DankDumpster** [comment]
@thesamesam sorry for mention but is the sign off correct?

**Comment by thesamesam** [comment]
Looks great now! Can test later :)

File: sys-fs/cryfs/cryfs-0.10.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+"
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}
+	test? ( dev-cpp/gtest )
+"
+
+PATCHES=(
+	# TODO upstream:
+	"${FILESDIR}/0.10.2-unbundle-libs.patch"
+	"${FILESDIR}/0.10.2-install-targets.patch"
```
Comment (thesamesam) [inline sys-fs/cryfs/cryfs-0.10.3.ebuild:41]: You forgot to test after changing ;)

You're missing a ${PN} here. It doesn't apply right now:
```
>>> Preparing source in /var/tmp/portage/sys-fs/cryfs-0.10.3/work ...
 * Applying 0.10.2-unbundle-libs.patch ...
/var/tmp/portage/sys-fs/cryfs-0.10.3/temp/environment: line 1611: /var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-unbundle-libs.patch: No such file or directory
/var/tmp/portage/sys-fs/cryfs-0.10.3/temp/environment: line 1614: /var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-unbundle-libs.patch: No such file or directory     [ !! ]
 * ERROR: sys-fs/cryfs-0.10.3::gentoo failed (prepare phase):
 *   patch -p1  failed with /var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-unbundle-libs.patch
 *
 * Call stack:
 *               ebuild.sh, line  125:  Called src_prepare
 *             environment, line 3516:  Called cmake_src_prepare
 *             environment, line 1528:  Called default_src_prepare
 *      phase-functions.sh, line  920:  Called __eapi6_src_prepare
 *             environment, line  237:  Called eapply '/var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-unbundle-libs.patch' '/var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-install-targets.patch'
 *             environment, line 1679:  Called _eapply_patch '/var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-unbundle-libs.patch'
 *             environment, line 1617:  Called __helpers_die 'patch -p1  failed with /var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-unbundle-libs.patch'
 *   isolated-functions.sh, line  112:  Called die
 * The specific snippet of code:
 *              die "$@"
 *
 * If you need support, post the output of `emerge --info '=sys-fs/cryfs-0.10.3::gentoo'`,
 * the complete build log and the output of `emerge -pqv '=sys-fs/cryfs-0.10.3::gentoo'`.
 * The complete build log is located at '/var/tmp/portage/sys-fs/cryfs-0.10.3/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/sys-fs/cryfs-0.10.3/temp/environment'.
 * Working directory: '/var/tmp/portage/sys-fs/cryfs-0.10.3/work'
 * S: '/var/tmp/portage/sys-fs/cryfs-0.10.3/work'
```

I'll fix

**Comment by thesamesam** [comment]
Cheers, I applied the GCC 11 patch too

**Pull request closed without merging**